<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1299</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>-3</y>
      <width>1301</width>
      <height>791</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>1011</width>
       <height>771</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>580</y>
        <width>459</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>If yes, then how much experience do you have? (Please select a value)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>If yes, then in which context did you use it?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>530</y>
        <width>401</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Do you have experience with VR (Virtual Reality) applications?</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>610</y>
        <width>611</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QRadioButton" name="radioButton_9">
         <property name="text">
          <string>1 (sehr schlecht)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_10">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_11">
         <property name="text">
          <string>3 (mittel)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_12">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_13">
         <property name="text">
          <string>5 (sehr gut)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>311</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
     <widget class="QPushButton" name="NextButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>590</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>0</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Demografischer Fragebogen</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>694</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Was für Sehfähigkeit haben Sie und welche Art? / Do you have normal or corrected vision and what type?</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>291</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>-Bitte auswählen- / -Please Select-</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>korrigierte Sehfähigkeit / corrected vision</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>unkorrigierte Sehfähigkeit / uncorrected vision</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sonstiges / Other</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_5">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>240</y>
        <width>341</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>-Bitte auswählen- / -Please Select-</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kurzsichtigkeit (Myopie) / nearsightedness (myopia)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Weitsichtigkeit (Hyperopie) / farsightedness (hyperopia)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Stabsichtigkeit (Astigmatismus) / astigmatism</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>perfektem Sehvermögen / perfect eyesight</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>441</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Have you had any experience with a haptic device prior to this study?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Do you have problems with tactile sensation?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Do you have problems with hearing?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>560</y>
        <width>459</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Wenn ja, wie viel Erfahrung haben Sie? (Bitte wählen Sie einen aus)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>630</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>(very little)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>630</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>(intermediate)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>630</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>(very good)</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>261</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Benutzername / Username:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_17">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>336</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Identifikationsnummer / ID Number:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>228</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_19">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>400</y>
        <width>754</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Haben Sie vor dieser Studie bereits Erfahrungen mit einem haptischen Gerät gemacht?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_5">
         <property name="text">
          <string>Ja / Yes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_6">
         <property name="text">
          <string>Nein / No</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>441</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Haben Sie Probleme mit dem Tastsinn?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_16">
         <property name="text">
          <string>Ja / Yes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_17">
         <property name="text">
          <string>Nein / No</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_21">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>290</y>
        <width>430</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Haben Sie Probleme mit dem Gehör?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_14">
         <property name="text">
          <string>Ja / Yes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_15">
         <property name="text">
          <string>Nein / No</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>561</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Wenn ja, in welcher Inhalt haben Sie es verwendet?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_23">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>510</y>
        <width>581</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Haben Sie Erfahrung mit VR (Virtual Reality)-Anwendungen?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_7">
         <property name="text">
          <string>Ja / Yes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_8">
         <property name="text">
          <string>Nein / No</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_24">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>180</y>
        <width>261</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Körpergröße / Height:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_25">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>180</y>
        <width>201</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_26">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Alter / Age:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_3"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_26">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>241</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Gewicht / Weight:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>110</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Geschlecht / Gender:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>103</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>F</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>103</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>M</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>140</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>R</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>140</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>L</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Links- oder Rechtshänder? / Left-handed or right-handed?</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>670</y>
        <width>531</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>561</width>
         <height>71</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
     <zorder>layoutWidget_15</zorder>
     <zorder>layoutWidget_16</zorder>
     <zorder>layoutWidget_17</zorder>
     <zorder>layoutWidget_18</zorder>
     <zorder>layoutWidget_19</zorder>
     <zorder>layoutWidget_20</zorder>
     <zorder>layoutWidget_21</zorder>
     <zorder>layoutWidget_22</zorder>
     <zorder>layoutWidget_23</zorder>
     <zorder>layoutWidget_24</zorder>
     <zorder>layoutWidget_25</zorder>
     <zorder>layoutWidget_26</zorder>
     <zorder>label_10</zorder>
     <zorder>label_8</zorder>
     <zorder>label_9</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>NextButton</zorder>
     <zorder>label_11</zorder>
     <zorder>label_13</zorder>
     <zorder>comboBox_4</zorder>
     <zorder>comboBox_5</zorder>
     <zorder>label_16</zorder>
     <zorder>label_17</zorder>
     <zorder>label_18</zorder>
     <zorder>label_21</zorder>
     <zorder>label_22</zorder>
     <zorder>label_23</zorder>
     <zorder>label_24</zorder>
     <zorder>label_3</zorder>
     <zorder>radioButton</zorder>
     <zorder>radioButton_2</zorder>
     <zorder>radioButton_4</zorder>
     <zorder>radioButton_3</zorder>
     <zorder>label_4</zorder>
     <zorder>frame_2</zorder>
    </widget>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
